### ISSUE TITLE
NoClassDefFoundError Using Proguard

### ISSUE THREAD

**Issue by carlyonj** [googlevr/gvr-android-sdk#538]
I am getting the error

` java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/vr/cardboard/VrParamsProviderFactory;`

Here are the vr dependencies I am including, and then I have extensive proguard rules, basically copying the one going around and adding to it extensively and nothing seems to fix this issue. I have tried multiple versions up to 120.

```
    implementation "com.google.vr:sdk-commonwidget:$googleVrVersion"
    implementation "com.google.vr:sdk-panowidget:$googleVrVersion"
    implementation "com.google.vr:sdk-videowidget:$googleVrVersion"
```

where I exclude 

```
      exclude group: 'com.google.android', module: 'aidl'
      exclude group: 'com.google.vr', module: 'sdk-common'
```

I thought this might be related to also including a pre 1.0 version of arCore but now that I am using arCore 1.0 I still get this identical error.

**Comment by carlyonj** [comment]
Doing that gives me the error 

`Duplicate zip entry [classes.jar:com/google/android/aidl/BaseProxy.class]`

which is probably because arCore and vr use this.

**Comment by jdduke** [comment]
We're pushing a fix for the aidl collision in the next release, which should be pushed tomorrow or Wednesday.

**Comment by sigmaxipi** [comment]
This has been fixed with v1.130.0.
